app/services/analytics.rb

```diff
@@ -248,6 +248,8 @@ def session_started_at
   USER_REGISTRATION_EMAIL_CONFIRMATION_RESEND = 'User Registration: Email Confirmation requested due to invalid token'
   USER_REGISTRATION_ENTER_EMAIL_VISIT = 'User Registration: enter email visited'
   USER_REGISTRATION_INTRO_VISIT = 'User Registration: intro visited'
+  USER_REGISTRATION_2FA_ADDITIONAL_SETUP = 'User Registration: Additional 2fa setup'
+  USER_REGISTRATION_2FA_ADDITIONAL_SETUP_VISIT = 'User Registration: Additional 2fa setup visited'
```

Comment:
could we we add these to `analytics_events.rb` as methods in the newer style so we can auto-generate documentation for them?